APMrover2/mode_auto.cpp

```diff
@@ -60,7 +60,7 @@ void ModeAuto::update()
             } else {
                 // we have reached the destination so stop
                 stop_vehicle();
-                lateral_acceleration = 0.0f;
+                _desired_lat_accel = 0.0f;
```

Comment:
Perhaps we could inform the controller here that we've become a bit apathetic about this whole movement thing?